experimental/packages/opentelemetry-instrumentation-grpc/test/helper.ts

```diff
@@ -505,12 +542,24 @@ export const runTests = (
       const validations = {
         name: `grpc.pkg_test.GrpcTester/${methodName}`,
         status: GrpcStatus.OK,
-        netPeerName: 'localhost',
-        netPeerPort: grpcPort,
+        host: 'localhost',
```

Comment:
Note for reviewer: This isn't a functional change, just resulted in less confusion on my end because "netPeerName" and "netPeerPort" are the literal values of the old attributes. host and port seemed more generic.